lib/membrane_live_audio_mixer.ex

```diff
@@ -0,0 +1,220 @@
+defmodule Membrane.LiveAudioMixer do
+  @moduledoc """
+  This element performs audio mixing.
+
+  Audio format can be set as an element option or received through stream_format from input pads. All
+  received stream_format have to be identical and match ones in element option (if that option is
+  different from `nil`).
+
+  Input pads can have offset - it tells how much timestamps differ from mixer time.
+
+  Mixer mixes only raw audio (PCM), so some parser may be needed to precede it in pipeline.
+  """
+
+  use Membrane.Filter
+  use Bunch
+
+  require Membrane.Logger
+
+  alias Membrane.AudioMixer.{Adder, ClipPreventingAdder, LiveQueue, NativeAdder}
+  alias Membrane.Buffer
+  alias Membrane.RawAudio
+  alias Membrane.Time
+
+  def_options stream_format: [
+                spec: RawAudio.t(),
+                description: """
+                The value defines a raw audio format of pads connected to the
+                element. It should be the same for all the pads.
+                """,
+                default: nil
+              ],
+              prevent_clipping: [
+                spec: boolean(),
+                description: """
+                Defines how the mixer should act in the case when an overflow happens.
+                - If true, the wave will be scaled down, so a peak will become the maximal
+                value of the sample in the format. See `Membrane.AudioMixer.ClipPreventingAdder`.
+                - If false, overflow will be clipped to the maximal value of the sample in
+                the format. See `Membrane.AudioMixer.Adder`.
+                """,
```

Comment:
What do we mean by an overflow?

Reply:
This option is taken from the regular audio mixer and the description is about how particular mixers work.